### PR TITLE
Stable composer configuration

### LINKED ISSUE #131: Help error composer install
Help error composer install.

 Problem 1
    - Installation request for doctrine/doctrine-mongo-odm-module dev-mast
er -> satisfiable by doctrine/doctrine-mongo-odm-module[dev-master].
    - doctrine/doctrine-mongo-odm-module dev-master requires doctrine/doct
rine-module 0.9.* -> no matching package found.

Thanks!

### PR DESCRIPTION
**Pull request by gianarb**
* Fixed #131
* Prepare this module for a stable release

### CODE REVIEW

**Comment by gianarb** [comment]
Reference https://github.com/ZF-Commons/ZfcUserDoctrineMongoODM/pull/20

File: composer.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     ],
     "require": {
         "php": ">=5.4",
-        "doctrine/doctrine-module": "0.9.*",
+        "doctrine/doctrine-module": ">=0.8.0",
```
Comment (Ocramius) [inline composer.json]: Use `~0.8`. This sort of constraint will go over 1.x and 2.x (and so on), likely causing issues

File: composer.json [hunk 2]
```diff
@@ -48,16 +48,10 @@
...
-    "minimum-stability": "dev",
     "autoload": {
         "psr-0": {
             "DoctrineMongoODMModule\\": "src/",
             "DoctrineMongoODMModuleTest\\": "tests/"
         }
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "0.9.x-dev"
```
Comment (Ocramius) [inline composer.json]: Only if you are tagging 0.9.0 afterwards

Comment (gianarb) [inline composer.json]: Yes after this merge I will tag it 0.9.0

**Pull request merged**